op-mode-definitions/crypt.xml.in

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="encryption">
+    <properties>
+      <help>Manage config encryption</help>
```

Comment:
Maybe mention the TPM part explicitly? `Manage configuration encryption using TPM`